cloud-azure/src/test/java/com/sequenceiq/cloudbreak/cloud/azure/AzureEncryptionResourcesTest.java

```diff
@@ -139,6 +142,40 @@ private void initActionFailedExceptionConversion() {
                 .thenAnswer(invocation -> new Retry.ActionFailedException(new CloudConnectorException(invocation.getArgument(0, Exception.class))));
     }
 
+    private static Object[][] getDesResourceId() {
+        return new Object[][]{
+                {"/subscriptions/dummySubscriptionId/resourceGroups/dummyAlphaResourceGroup/providers/" +
+                        "Microsoft.Compute/diskEncryptionSets/dummyDes",
+                        "dummyAlphaResourceGroup"},
+                {"/subscriptions/dummySubscriptionId/resourceGroups/dummyAlphaNumResourceGroup0910/providers/" +
+                        "Microsoft.Compute/diskEncryptionSets/dummyDes",
+                        "dummyAlphaNumResourceGroup0910"},
+                {"/subscriptions/dummySubscriptionId/resourceGroups/dummyAlphaNumHyphenResourceGroup---0910/providers/" +
+                        "Microsoft.Compute/diskEncryptionSets/dummyDes",
+                        "dummyAlphaNumHyphenResourceGroup---0910"},
+                {"/subscriptions/dummySubscriptionId/resourceGroups/dummyAlphaNumHyphenPeriodResourceGroup---....0910/providers/" +
+                        "Microsoft.Compute/diskEncryptionSets/dummyDes",
+                        "dummyAlphaNumHyphenPeriodResourceGroup---....0910"},
+                {"/subscriptions/dummySubscriptionId/resourceGroups/dummyAlphaParenthesesPeriodResourceGroup---....)()(0910/providers/" +
+                        "Microsoft.Compute/diskEncryptionSets/dummyDes",
+                        "dummyAlphaParenthesesPeriodResourceGroup---....)()(0910"},
+                {"/subscriptions/dummySubscriptionId/resourceGroups/dummyUnderscroceResourceGroup---___....)()(0910/providers/" +
+                        "Microsoft.Compute/diskEncryptionSets/dummyDes",
+                        "dummyUnderscroceResourceGroup---___....)()(0910"}
+        };
+    }
+
+    @ParameterizedTest
+    @MethodSource("getDesResourceId")
+    public void testDiskEncryptionSetResourceGroupNameExtraction(String desResourceId, String resourceGroupName) {
```

Comment:
- To make it easier to identify the individual test cases of a parameterized test, it is a convention to add some "test case description". This is done with `@ParameterizedTest(name = "{0}")` where the value of the `name` property can contain placeholders referencing method arguments; `{0}` is the first (leftmost) one. See https://github.com/hortonworks/cloudbreak/blob/master/environment-api/src/test/java/com/sequenceiq/environment/api/v1/environment/model/request/azure/AzureResourceEncryptionParametersTest.java#L79-L81 for an example. Either add a dedicated argument like `testCaseName` there, or simply reference one or more arguments that are used to construct the test case description.
- Again, as a convention, method arguments denoting expected values should be named accordingly. So `resourceGroupName` should be renamed to something like `resourceGroupNameExpected` or `expectedResourceGroupName`, whichever you prefer.